crates/db_schema/src/source/images.rs

```diff
@@ -56,3 +58,28 @@ pub struct RemoteImage {
 pub struct RemoteImageForm {
   pub link: DbUrl,
 }
+
+#[skip_serializing_none]
+#[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "full", derive(Queryable, Selectable, Identifiable, TS))]
+#[cfg_attr(feature = "full", ts(export))]
+#[cfg_attr(feature = "full", diesel(table_name = image_details))]
+#[cfg_attr(feature = "full", diesel(check_for_backend(diesel::pg::Pg)))]
+#[cfg_attr(feature = "full", diesel(primary_key(link)))]
+pub struct ImageDetails {
+  pub link: DbUrl,
+  pub width: i32,
+  pub height: i32,
+  pub content_type: String,
+  pub published: DateTime<Utc>,
```

Comment:
`RemoteImage` and `LocalImage` already have a timestamp which should be identical, so this is useless.

Reply:
K, I'll remove.